test/io-tests/test_io.py

```diff
@@ -283,6 +283,24 @@ def test_expected_config_from_db_settings(defaultenv):
 
     assert dumpconfig(configpath=config, env=env) == expected
 
+def test_expected_config_from_db_settings_other_authenticator(defaultenv):
+    "NOTIFY reload config should show an error if role-claim-key is invalid"
```

Comment:
The description seems wrong. Also the `PGRST_DB_CHANNEL_ENABLED` looks like this test was copied from somewhere ;).

I suggest to just parametrize the `test_expected_config_from_db_settings` test with a db role parameter. Then this will run twice with just a different user.

Set env to something like:
```python
db_uri = defaultenv["PGRST_DB_URI"].replace("user=postgrest_test_authenticator", f"user={role}")
[...]
  "PGRST_DB_URI": db_uri
```
(in the regular case this will just replace `postgrest_test_authenticator` with `postgrest_test_authenticator`)

And then load the the expected config file from either by using multiple parameters with `@pytest.mark.parametrize` or by putting the role name into the filename.

Reply:
Added the pytest parameter. Thanks a lot for your help Wolfgang :). I'm still getting used to the python tests.